package-lock.json

```diff
@@ -0,0 +1,14217 @@
+{
```

Comment:
Please use yarn rather than npm to add packages - then there shouldn't be any package-long.json and yarn.lock will update